install/kubernetes/helm/meshery/charts/meshery-osm/templates/NOTES.txt

```diff
@@ -17,5 +17,5 @@
 {{- else if contains "ClusterIP" .Values.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "meshery-osm.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
   echo "Visit http://127.0.0.1:8080 to use your application"
-  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:80
+  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:10009
```

Comment:
Why we not use {{ .Values.service.port }} to populate value of port "10009". This could be used for all port and in this way we will have only one place where they have to be changed (values.yaml), because from what I noticed we changed them manually here in NOTES.txt

Reply:
`port-forward` command forward the service which in the pod, we should not use the `service` port and we need to use the `service` port.

Reply:
That's may good point is here that we can change the service port which in the pod into `values.yaml`. I will do this in next PR.

Reply:
Yes, my point is to point it to the correct port from the values.yaml

Reply:
I mean we add the adapter pod port into values.yaml, not mean use the service port of the pod. That's two different things.